wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -73,8 +74,8 @@ namespace :wagn do
 
   desc "migrate structure and cards"
   task :migrate =>:environment do
-    ENV['SCHEMA'] ||= "#{Wagn.root}/db/schema.rb"
-    
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
```

Comment:
Then I guess all of these should be like this one?  If schema is always in the gem.  I think that is one thing that will change a little to complete the generator work.  It may be that for the cards in another rails app case we have to extend the rails command vs. having our own (wagn or decko).  In other words, we'd have both, one for the current case of the app (wagn) and in decko-rails we extend the rails command for the new case.